homeworks/magurmarian/hw01/test_calculation.py

```diff
@@ -11,17 +12,18 @@
 
 
 class SmokeTest(Testcase):
-
+    """class for smoke testing add and devide functions"""
     @test
-    def test_add(self, steps, num1, num2):
-        with steps.start('Result bigger then 0') as step:
-            if add(num1, num2) > 0:
-                assert add(num1, num2) > 0
-            else:
-                self.skipped('Result is less then 0)')
+    def test_add(self, num1, num2):
+        """method for testing add function"""
+        if add(num1, num2) >= 0:
```

Comment:
Excessive code. Simply skipping, when function results in negative value, will be enough.